R/ft_get_si.R

```diff
@@ -0,0 +1,240 @@
+#' Download supplementary materials from journals
```

Comment:
A `@details` section might be nice in this man file to explain this function a bit, and to state what publishers are available via this fxn, and what will come down the line